src/main/java/com/parasoft/demoapp/model/industry/ItemInventoryEntity.java

```diff
@@ -0,0 +1,21 @@
+package com.parasoft.demoapp.model.industry;
+
+import lombok.*;
+
+import javax.persistence.*;
+
+@Getter
+@Entity
+@Table(name="tbl_item_inventory")
+@NoArgsConstructor
+@EqualsAndHashCode
+@ToString
+public class ItemInventoryEntity {
+    @Id
+    @Column(name = "item_id")
```

Comment:
I see for id in other entities, we have annotation: @GeneratedValue(strategy= GenerationType.IDENTITY).
I searched online seems for HSQL it will pick INDENTITY as default so the code is fine. 
Maybe just for consistency let's add the annotation here.

Reply:
The value of column "item_id" in item inventory table is not generated by DB or JPA. It's from the item table. So I think it is not necessay to add @GeneratedValue(strategy= GenerationType.IDENTITY) annotation.